pkg/api/unversioned/group_version.go

```diff
@@ -179,6 +179,18 @@ func (gv GroupVersion) String() string {
 	return gv.Version
 }
 
+// VersionForGroup implements the same interface as GroupVersions to allow easy
+// selection of group versions.
+func (gv GroupVersion) VersionForGroup(group string) (GroupVersion, bool) {
```

Comment:
per our conversation, I think this needs to take a `GroupKind` instead of just a `Group`, since different Kinds inside a Group may choose to serialize differently.